src/main/markdown/lifeofanissue.md

```diff
@@ -12,9 +12,9 @@ One of the best ways you can help us improve GWT is to let us know about any pro
 
 Here's how to report issues:
 
-1.  [Search for your issue](http://code.google.com/p/google-web-toolkit/issues/advsearch) to see if anyone has already reported it.
+1.  [Search for your issue](https://github.com/gwtproject/gwt/issues?q=is%3Aissue) to see if anyone has already reported it.
 2.  If you find your issue and it's important to you, star it. That's how we know which issues are most important to fix, and you'll get notified of new activity on the issue.
-3.  If no one's reported your issue, [file it](http://code.google.com/p/google-web-toolkit/issues/entry). Please fill out the template completely. It is especially important to provide browser version(s) and a small code sample which can be used to reproduce the issue.
+3.  If no one's reported your issue, [file it](https://github.com/gwtproject/gwt/issues/new). Please provide the version of GWT, the browser(s), and platform(s) you're using. It is especially important to provide the browser version(s) and a small code sample which can be used to reproduce the issue. If the issue has been discussed in forums, please provide a link to the discussion too.
```

Comment:
Okay, if you are dragging me into this, I'd tend to say avoid 'you' when talking about the bug, since its more about where the issue can be reproduced than specific what you use.

But with that said, both 'sound' correct to me (as a native English speaker). Checking with the writing team in my office, 'versions' is more correct, unless we expect GWT, platform and browser to have the same versions.